spec/models/grda_warehouse/hud/project_spec.rb

```diff
@@ -41,7 +41,13 @@
   let!(:pc5) { create :hud_project_coc, data_source_id: ds2.id, ProjectID: p7.ProjectID, CoCCode: 'foo', hud_coc_code: 'bar' }
   let!(:pc6) { create :hud_project_coc, data_source_id: ds2.id, ProjectID: p8.ProjectID, hud_coc_code: 'bar' }
 
-  u = ->(user) { model.viewable_by(user).pluck(:id).sort }
+  u = ->(user) do
+    if model == GrdaWarehouse::Hud::Project
+      model.viewable_by(user, project_scope: :all).pluck(:id).sort
+    else
+      model.viewable_by(user).pluck(:id).sort
+    end
+  end
```

Comment:
Can you also update [this test](https://github.com/greenriver/hmis-warehouse/blob/5846f26291a9e446605c1378360b6a4f46137423/spec/models/grda_warehouse/hud/project_spec.rb#L140) to test the updated `viewable_by` scope?